### ISSUE TITLE
custom importers and exporters

### ISSUE THREAD

**Issue by antialias** [jsonresume/resume-cli#356]
Similar to how themes are implemented as separate packages, it would be nice to have community-developed importers and exporters in order to implement interoperability with other resume formats, e.g. h-resume and europass.

Related ticket: https://github.com/jsonresume/resume-schema/issues/280

**Comment by evanplaice** [comment]
Closed as duplicate of #357

**Comment by antialias** [comment]
Can we keep this open? It's a different ask. This issue describs adding custom importers and exporters to resume-cli whereas #357 requests adding support for Europass specifically.

**Comment by evanplaice** [comment]
Yes, just try to direct the conversation toward actionable changes. I'm trying my best to clean up a massive backlog of issues that were either not actionable or didn't result in any meaningful work.

**Comment by antialias** [comment]
> Yes, just try to direct the conversation toward actionable changes.

This is pretty much done in my branch. I mostly opened this ticket to get feedback from the dev community (i.e. you as far as I can tell, lol) before opening a PR.

**Comment by antialias** [comment]
(btw, I have addressed all the open and still relevant issues for resume-cli in my own fork; kinda waiting for feedback on my first PR before opening the next one though)

**Comment by thomasdavis** [comment]
Made a "Request For Comment" label to avoid stepping on toes. 

Essentially we should just be adding all the importers and exporters to the homepage, I think there are quite a few already.

https://github.com/joshuatz/linkedin-to-jsonresume

**Comment by thomasdavis** [comment]
Instead of bringing importers/exporters directly into jsonresume code. 

I suggest we have a naming convention much like themes. 

`jsonresume-importer-europass` 
`jsonresume-exporter-europass` 

Then given enough traction we support a command such as 

`resume import europass` (something else probably)

**Comment by antialias** [comment]
The codebase is ready for this feature to be added. I will start work this weekend or Monday.

**Comment by thomasdavis** [comment]
@antialias custom importers/exporters was a thoughtful RFC — too large for an archived repo, but the monorepo now has a `packages/converters` area where this thinking is directly relevant. Would love to see it re-proposed there.

Status update: **JSON Resume is under active maintenance again (2026)**, and resume-cli has been revived — development now happens in the [jsonresume.org monorepo](https://github.com/jsonresume/jsonresume.org) at [`packages/cli`](https://github.com/jsonresume/jsonresume.org/tree/master/packages/cli): Node 18+ support, the install-breaking husky hook removed, puppeteer bumped to `^23`, tests green. The npm package name `resume-cli` is unchanged. Full announcement: #786 · Consolidation tracking: [jsonresume/jsonresume.org#275](https://github.com/jsonresume/jsonresume.org/issues/275).

This standalone repo is being archived with a redirect, so I'm closing this out as part of the migration triage. If this still affects you on the revived version, please open a fresh issue in the monorepo where it will actually get eyes: https://github.com/jsonresume/jsonresume.org/issues. Thanks for taking the time to report it — and sorry it sat unanswered for so long.
